runtime/doc/builtin.txt

```diff
@@ -1857,10 +1857,15 @@ complete_info([{what}])				*complete_info()*
 				See |complete_info_mode| for the values.
 		   pum_visible	|TRUE| if popup menu is visible.
 				See |pumvisible()|.
-		   items	List of completion matches.  Each item is a
-				dictionary containing the entries "word",
+		   items	List of all completion candidates. Each item
+				is a dictionary containing the entries "word",
 				"abbr", "menu", "kind", "info" and "user_data".
 				See |complete-items|.
+		   matches	Same as "items", but only returns items that
+				are shown in the |popup-menu|. If both "matches"
+				and "items" are in "what", the returned list
+				will still be named "items", but each item
+				will have an additional "match" field.
```

Comment:
The `match` field is not documented in `:h complete-items`, so its type is not specified.

Reply:
Yes. Because it only exists if what contains matches. So there is no need to record it there...

Reply:
But the _type_ of the field is not specified, even here.

If you simply do `:help complete-items`, which I do _regularly_, you won't see all of the possible keys in that list. And seeing an undocumented `match` field (e.g. in some test log or whatever) would look like a bug.

I suggest that it is added to the documentation at `:help complete-items` with a reference to `|complete_info|`

Reply:
okay  added.